### PR TITLE
Release 0.8.1 changes

### PR DESCRIPTION
**Pull request by Clintonio**
This branch will not actually be merged, here for review only.

Check out the commits only, the full diff is too much.

### CODE REVIEW

File: docs/source/release-notes.rst [hunk 1]
```diff
@@ -3,6 +3,12 @@ Release notes
 
 Here are brief summaries of what's changed between each snapshot release.
 
+Milestone 8.1
+-------------
+
+* Fix: The race condition causing webservers to crash during startup
```
Comment (adagys) [inline docs/source/release-notes.rst]: end with . for consistency

Comment (Clintonio) [inline docs/source/release-notes.rst]: Fixed.

**Comment by adagys** [comment]
assuming the fixes were already reviewed and tested, looks ok. With the .1 release do we still refer to M8 as being the latest release in the docs? we probably have to update any references to the release tag

**Comment by Clintonio** [comment]
Good point. Found every reference I could.

File: docs/source/getting-set-up-fault-finding.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@ Milestone releases
 
 When you clone the corda or cordapp-template repos, they will default to the master branch. The master branch is being continuously developed upon, and its features may not align with the state of Corda as described in the docs. Additionally, the master branch of the CorDapp Template may break in response to changes in the main corda repo.
 
-When developing on Corda, you should always check out the latest stable branch instead, by running ``git checkout release-M7``.
+When developing on Corda, you should always check out the latest stable branch instead, by running ``git checkout release-M8.1``.
```
Comment (adagys) [inline docs/source/getting-set-up-fault-finding.rst]: does this refer to tag or branch? I'm assuming we're going to create a new tag, but keep the M8 branch?

Comment (Clintonio) [inline docs/source/getting-set-up-fault-finding.rst]: I'm going to change that text to refer to the tag. We shouldn't be suggesting the branches.

**Pull request merged**